phpdotnet/phd/Package/PHP/XHTML.php

```diff
@@ -442,7 +442,7 @@ public function format_methodsynopsis($open, $name, $attrs, $props) {
         if ($this->cchunk["methodsynopsis"]["returntypes"]) {
             $types = [];
             foreach ($this->cchunk["methodsynopsis"]["returntypes"] as $return_type) {
-                $formatted_type = self::format_type_if_object_or_pseudo_text($return_type, "type");
```

Comment:
I believe `phpdotnet\phd\Package_PHP_XHTML::format_type_if_object_or_pseudo_text` is not used anywhere else so itcan be removed

Reply:
There's still another problem with simple nullable parameters:
```
/* Methods */
public __construct(
    ? $locale,
    int $dateType,
    int $timeType,
    IntlTimeZone|DateTimeZone|string|null $timezone = null,
    IntlCalendar|int|null $calendar = null,
    string $pattern = ""
)
```